### ISSUE TITLE
Change version of "mdi" in "bower.json" file.

### ISSUE THREAD

**Issue by alex-oliveira** [lumapps/lumX#394]
I think that would be better set the version of "mdi" like "latest" and you?

**Comment by alex-oliveira** [comment]
I noted that the version was fixed to correct a problem of ui.
https://github.com/lumapps/lumX/commit/d1c29eb12394619fc2bdf524dfb241281330e1eb

This change make us miss many new icons.
I think that a better solution would create a css file to each new version of mdi.
I think that is better that stay outdated.
I created a css file to resolve the problems with the version ~1.4.57 of mdi:

`````
/* "MaterialDesignIcons" changed to "Material Design Icons" then  */
.checkbox__label:after, .checkbox__label:before,
.data-table__selectable-row td:first-child:before, .data-table__selectable-row th:first-child:before,
.data-table__sortable-cell:before,
.radio-button__label:after, .radio-button__label:before,
.lx-select__selected:after,
.lx-select__choice--is-multiple:after, .lx-select__choice--is-multiple:before,
.lx-select__delete-button {
    font: normal normal normal 24px / 1 "Material Design Icons";
}

/* "\f203" is same to ".mdi-checkbox-blank-outline:before" then  */
.checkbox__label:before,
.lx-select__choice--is-multiple:before,
.data-table__selectable-row td:first-child:before, .data-table__selectable-row th:first-child:before {
    content: "\f227" !important;
}

/* "\f204" is same to ".mdi-checkbox-marked:before" then */
.checkbox__label:after,
.lx-select__choice--is-multiple:after,
.data-table__selectable-row--is-selected td:first-child:before, .data-table__selectable-row--is-selected th:first-child:before {
    content: "\f228" !important;
}

/* "\f14c" is same to ".mdi-arrow-up:before" then */
.data-table__sortable-cell--asc:before {
    content: "\f156" !important;
}

/* "\f13a" is same to ".mdi-arrow-down:before" then */
.data-table__sortable-cell--desc:before {
    content: "\f144" !important;
}

/* "\f4b1" is same to ".mdi-radiobox-blank:before" then */
.radio-button__label:before {
    content: "\f52c" !important;
}

/* "\f4b2" is same to ".mdi-radiobox-marked:before" then */
.radio-button__label:after {
    content: "\f52d" !important;
}

/* "\f3f2" is same to ".mdi-menu-down:before" then */
.lx-select__selected:after {
    content: "\f44f" !important;
}

/* "\f223" is same to ".mdi-close:before" then */
.lx-select__delete-button:after {
    content: "\f24c" !important;
}`
`````

**Comment by matmkian** [comment]
We've just released LumX v1.0.0rc1. This new version fixes the issue, so I'm closing it.
